### PR TITLE
Hearing Worksheet- fix pageload error

### PR DESCRIPTION
**Pull request by nolawi**
#3548 

- Adds alert message to both dockets and hearings
- detects if `serverError` and displays alert

### CODE REVIEW

File: client/app/hearings/DailyDocketContainer.jsx [hunk 1]
```diff
@@ -21,8 +22,12 @@ export class DailyDocketContainer extends React.Component {
 
   render() {
     if (this.props.serverError) {
-      return <div style={{ textAlign: 'center' }}>
-        An error occurred while retrieving your hearings.</div>;
+      return <div className="cf-app-segment cf-app-segment--alt cf-hearings">
+        <Alert
+          title="Error Status"
+          type="error"> An error occurred while retrieving your Daily Docket.
+        </Alert>
```
Comment (sharonwarner) [inline client/app/hearings/DailyDocketContainer.jsx]: Can we provide a suggestion as well? Maybe 'Try refreshing the page.'

File: client/app/hearings/HearingWorksheetContainer.jsx [hunk 2]
```diff
@@ -52,8 +57,8 @@ export class HearingWorksheetContainer extends React.Component {
 }
 
 const mapStateToProps = (state) => ({
-  // TODO: add mappings
-  worksheet: state.worksheet
+  worksheet: state.worksheet,
+  serverError: state.serverError
```
Comment (sharonwarner) [inline client/app/hearings/HearingWorksheetContainer.jsx]: I think this should be specific for the page. We don't want the Daily Docket to show an error if it's just the Hearing Worksheet request that failed.

Comment (nolawi) [reply client/app/hearings/HearingWorksheetContainer.jsx]: Agree, the worksheet will load even if dockets fails and vice versa. It's the same `handleServerError` action but it works as you described it.

Comment (sharonwarner) [reply client/app/hearings/HearingWorksheetContainer.jsx]: We can talk after standup about it, but both pages share the same serverError field. So they'll both display errors at the same time.

File: client/app/hearings/DailyDocketContainer.jsx [hunk 3]
```diff
@@ -59,17 +65,17 @@ export class DailyDocketContainer extends React.Component {
...
   dockets: state.dockets,
-  serverError: state.serverError
+  docketServerError: state.docketServerError
 });
 
 const mapDispatchToProps = (dispatch) => ({
   getDockets: (dockets) => () => {
     if (!dockets) {
-      ApiUtil.get('/hearings/dockets.json', { cache: true }).
+      ApiUtil.get('/hearings/docketsd.json', { cache: true }).
```
Comment (sharonwarner) [inline client/app/hearings/DailyDocketContainer.jsx]: Accidental commit?

Comment (nolawi) [reply client/app/hearings/DailyDocketContainer.jsx]: yap let me fix

**Pull request merged**